arches_lingo/src/arches_lingo/components/scheme/report/SchemeSection.vue

```diff
@@ -2,6 +2,8 @@
 import { useGettext } from "vue3-gettext";
 import Button from "primevue/button";
 
+defineEmits(["openEditor"]);
```

Comment:
nit: declare string as constant

Reply:
not done, per https://github.com/archesproject/arches-lingo/pull/124#discussion_r1853305909